jsonschema/validate.go

```diff
@@ -0,0 +1,89 @@
+package jsonschema
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+func Unmarshal(schema Definition, content []byte, v any) error {
```

Comment:
```suggestion
func VerifySchemaAndUnmarshal(schema Definition, content []byte, v any) error {
```

It would be great if we could separate all validation stuff into a separate PR — I'm struggling to review this due to the large size.